web/src/drafts.js

```diff
@@ -548,8 +548,8 @@ const keyboard_handling_context = {
     },
     items_container_selector: "drafts-container",
     items_list_selector: "drafts-list",
-    row_item_selector: "draft-row",
-    box_item_selector: "draft-info-box",
+    row_item_selector: "overlay-message-row",
+    box_item_selector: "overlay-message-info-box",
```

Comment:
So I worry that these changes may result in problems where the logic 

@amanagr does `messages_overlay_ui.ts` need a small refactor to ensure that it's always using these within the scope of say `drafts-list`? A lot of the selectors, like this function, do not have an obvious mechanism through which they wouldn't accidentally match the similar elements in the scheduled messages overlay.

```
export function row_with_focus(context: Context): JQuery {                                              
    const focused_item = $(`.${CSS.escape(context.box_item_selector)}:focus`)[0];                       
    return $(focused_item).parent(`.${CSS.escape(context.row_item_selector)}`);                         
}                                                                                                       
```